### ISSUE TITLE
[Request] YTLite 5.0 dropped, please update to it

### ISSUE THREAD

**Issue by reticivis-net** [YTLitePlus/YTLitePlus#394]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you use YouTube Premium?

✅ Yes, I do use YouTube Premium

### My feature request

https://github.com/dayanch96/YTLite/releases/tag/v5.0

please update ytliteplus to use 5.0

**Comment by bhackel** [comment]
Working on removing duplicates, will be ready soon

**Comment by KieranBLK** [comment]
💀 i just realized YTLitePlus isn't managed by Dan. Lmao. 🤣

**Comment by bhackel** [comment]
> 💀 i just realized YTLitePlus isn't managed by Dayanch. Lmao. 🤣

Yeah the branding is confusing. YTLitePlus is an extension of YouTube Plus (formerly known as YTLite) with a focus on making a sideload-able iPA with many other tweaks.

**Comment by HyperrJedi** [comment]
Will sponsorblock be removed on ytliteplus since YouTube plus has in now?

**Comment by bhackel** [comment]
iSponsorBlock will be kept but set to default disabled. Then the user can pick which one they prefer. This is because the current implementation of SponsorBlock in YouTube Plus does not have every feature that people want.
